### PR TITLE
feat: Add Skiff to app store

### PR DESCRIPTION
**Pull request by amilich**
## What does this PR do?

Now that Skiff Calendar works with Cal.com, this PR adds Skiff as a link app to the app store.

Fixes # (issue)

## Requirement/Documentation

  - If there is a requirement document, please, share it here.
  - If there is ab UI/UX design document, please, share it here.

## Type of change

  - [ ] Bug fix (non-breaking change which fixes an issue)
  - [ ] Chore (refactoring code, technical debt, workflow improvements)
  - [x] New feature (non-breaking change which adds functionality)
  - [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
  - [ ] This change requires a documentation update

## How should this be tested?

  - Are there environment variables that should be set?

Simply run the app store locally.

  - What are the minimal test data to have?

None needed.

  - What is expected (happy path) to have (input and output)?

See image and video below.

  - Any other important info that could help to test that PR

No other info needed.

## Mandatory Tasks

  - [x] Make sure you have self-reviewed the code. A decent size PR without self-review might be rejected.

## Checklist

- I haven't read the [contributing guide](https://github.com/calcom/cal.com/blob/main/CONTRIBUTING.md)
- My code doesn't follow the style guidelines of this project
- I haven't commented my code, particularly in hard-to-understand areas
- I haven't checked if my PR needs changes to the documentation
- I haven't checked if my changes generate no new warnings
- I haven't added tests that prove my fix is effective or that my feature works
- I haven't checked if new and existing unit tests pass locally with my changes

### CODE REVIEW

**Comment by PeerRich** [comment]
hell yeah!! lets go

**Comment by PeerRich** [comment]
can you explain more how this integration works? as far as i can see it just links to skiff, is this correct? is there plans for OAuth or so?

**Comment by amilich** [comment]
> can you explain more how this integration works? as far as i can see it just links to skiff, is this correct? is there plans for OAuth or so?

Definitely. The plan is to build a similar link-type setting inside the Skiff Calendar settings page (see https://app.skiff.com/calendar) and then add an API availability integration.

**Comment by amilich** [comment]
Screenshot 2023-07-12 at 4 03 07 PM

Uploading Screen Recording 2023-07-12 at 4.04.31 PM.mov…

**Comment by PeerRich** [comment]
ok, how does skiff connect to cal now? I mean, how do the events go into the calendar? is there a caldav interface? If yes, then instead of linkning to the install page, why dont we use the CalDav code?

**Comment by PeerRich** [comment]
or is there a blog post explaining how  to connect? if yes, then lets link there vs. just the landing page

**Comment by PeerRich** [comment]
awesome! i think we can merge this and then have a follow up PR once the OAuth / API solution is ready

**Review by PeerRich — DISMISSED** [review]

File: packages/app-store/skiff/components/.gitkeep
Comment (PeerRich) [inline packages/app-store/skiff/components/.gitkeep:1]: i dont think this has to be commited

File: yarn.lock
Comment (PeerRich) [inline yarn.lock:1]: can you uncommit yarn lock too?

Comment (amilich) [reply yarn.lock:1]: yep! doing all this now

File: packages/app-store/skiff/components/.gitkeep
Comment (amilich) [reply packages/app-store/skiff/components/.gitkeep:1]: sgtm

**Comment by amilich** [comment]
@PeerRich just resolved comments. LMK if anything else I should do :)

**Review by PeerRich — APPROVED** [review]

**Pull request merged**